electron-env

```diff
@@ -9,15 +9,19 @@
 # python-qt and its dependencies will still need to be installed with
 # your package manager.
 
+# System dependencies
+# sudo apt-get install python3-virtualenv python3-pip virtualenv git libsecp256k1-0
+
 if [ -e ./env/bin/activate ]; then
     source ./env/bin/activate
 else
     virtualenv env -p `which python3`
     source ./env/bin/activate
     python3 setup.py install
+    pip3 install -I pyqt5
 fi
 
-export PYTHONPATH="/usr/local/lib/python3.5/site-packages:$PYTHONPATH"
+#export PYTHONPATH="/usr/local/lib/python3.5/site-packages:$PYTHONPATH"
```

Comment:
Any reason to leave this? I think it already does nothing.

Reply:
I'm imagining because python 3.6 is what's more common now and that path is python3.5? (Just guessing here..)

Reply:
My preference is just to remove dead / old code.

Reply:
Oh, yeah I get what you were asking now.  Yeah for sure.